src/backend/distributed/shared_library_init.c

```diff
@@ -848,6 +849,16 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
```

Comment:
I think we should find a more specific name than `disable_citus_preconditions` but I'll think about that later.

Do you have any other suggestions?